src/GT-Playground/GTPlayBook.class.st

```diff
@@ -26,6 +26,7 @@ Class {
 GTPlayBook class >> cacheDirectory [
 	" lazy initialize the cache and always ensure its existence - especially while the image is running "
 	cacheDirectory ifNil: [ cacheDirectory := self defaultCacheDirectory ].
+	cacheDirectory fileSystem isWritable ifFalse: [ ^ cacheDirectory ].
```

Comment:
Why you do not use existing fileReference API isReadable/isWritable?
We should not expose internals of fileReference.

Reply:
They of course require some changes to support "forced readOnly file systems"

Reply:
Following and improving real FileReference API will benefit us to fix installation to really read only OS directories

Reply:
The problem in some usages of the FileReference is that isWritable on has different semantics. It requires an existing referenced file while we need to check if we will be able to create such file/directory. Of course, a proper method in FileReference API would be better (canBeCreated?)

Reply:
Yes for #canBeCreated